mitosheet/css/taskpanes/Graph/GraphSidebar.css

```diff
@@ -93,6 +110,45 @@
     text-align: center;
 }
 
+.graph-element-popup-div.popup-div:before {
+    content: "";
+    display: block;
+    height: 10px;
+    width: 10px;
```

Comment:
Looking at the graph titles of the graphs in the shaping document, they are usually longer than can fit in this input currently. 

I think we should make two updates:

1. When we open the title editor, have it long enough to display the current title (obviously up to certain extent if the column header is super long)
2. If the user adds more text to the label, have the text input expand (we do this for the cell editor in the grid so there is probably a utility you can use)

Reply:
Just updated! I refactored / moved the util you were talking about to be more general and it's working well! Lmk if you want me to adjust any of the defaults (max or min length)

Reply:
Looks good!